apps/pie-microsite/src/utilities/helpers.js

```diff
@@ -27,12 +31,18 @@ const capitaliseFirstLetter = string => {
     return capitalised;
 };
 
+// checks whether value can convert to number
```

Comment:
Can we use JS docs here rather than double slash? See the other functions in this file for examples